ExoPlayer playback fails when attempting to play multiple audio tracks, transcoding issues

### Describe the bug

Upgraded to v2.4, seems that some H264/DTS5.1 media are not playing. Another case is, it will buffer for a second and get stuck at pause state and can never be played. 

If these media is then stuck at pause state or not playing, changing the audio/subtitle will lead for the app to crash.

There are no errors stated in the app, it is just stuck at pause state (black screen).

Here are some media that experienced this issue so far:
    
- Pirates of the Caribbean The Curse of the Black Pearl (2003) [tmdb-22][Bluray-1080p][8bit][x264][DTS 5.1]-TayTO.mkv
- Pirates of the Caribbean Dead Mans Chest (2006) [tmdb-58][Bluray-1080p][8bit][x264][DTS 5.1]-Z0N3.mkv
- Pirates of the Caribbean On Stranger Tides (2011) [tmdb-1865][Bluray-1080p][8bit][x264][DTS 5.1]-TayTO.mkv

In v2.3.3 these files are playing properly wherein their audio are transcoded into AAC.
Other H264/DTS5.1 media plays properly in v2.4.





### Logs


[logcat.txt](https://github.com/jellyfin/jellyfin-android/files/7487719/logcat.txt)

### Application version

2.4.0

### Where did you install the app from?

Google Play

### Device information

Samsung A10, LG G6

### Android version

Android 9, Android 8

### Jellyfin server version

10.7.7

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [X] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

```
...
11-06 00:06:12.641 18798 18798 E EventLogger: playerFailed [eventTime=1.91, mediaPos=0.00, window=0, period=0, errorCode=ERROR_CODE_UNSPECIFIED
11-06 00:06:12.641 18798 18798 E EventLogger:   com.google.android.exoplayer2.ExoPlaybackException: Unexpected runtime error
11-06 00:06:12.641 18798 18798 E EventLogger:       at com.google.android.exoplayer2.DefaultMediaClock.onRendererEnabled(DefaultMediaClock.java:96)
11-06 00:06:12.641 18798 18798 E EventLogger:       at com.google.android.exoplayer2.ExoPlayerImplInternal.enableRenderer(ExoPlayerImplInternal.java:2447)
11-06 00:06:12.641 18798 18798 E EventLogger:       at com.google.android.exoplayer2.ExoPlayerImplInternal.enableRenderers(ExoPlayerImplInternal.java:2395)
11-06 00:06:12.641 18798 18798 E EventLogger:       at com.google.android.exoplayer2.ExoPlayerImplInternal.reselectTracksInternal(ExoPlayerImplInternal.java:1702)
11-06 00:06:12.641 18798 18798 E EventLogger:       at com.google.android.exoplayer2.ExoPlayerImplInternal.handleMessage(ExoPlayerImplInternal.java:507)
11-06 00:06:12.641 18798 18798 E EventLogger:       at android.os.Handler.dispatchMessage(Handler.java:102)
11-06 00:06:12.641 18798 18798 E EventLogger:       at android.os.Looper.loop(Looper.java:214)
11-06 00:06:12.641 18798 18798 E EventLogger:       at android.os.HandlerThread.run(HandlerThread.java:65)
11-06 00:06:12.641 18798 18798 E EventLogger:   Caused by: java.lang.IllegalStateException: Multiple renderer media clocks enabled.
11-06 00:06:12.641 18798 18798 E EventLogger:       ... 8 more
11-06 00:06:12.641 18798 18798 E EventLogger: ]
...
11-06 00:06:12.642 18798 18798 D EventLogger:   MediaCodecAudioRenderer [
11-06 00:06:12.642 18798 18798 D EventLogger:     Group:0, adaptive_supported=N/A [
11-06 00:06:12.642 18798 18798 D EventLogger:       [X] Track:0, id=3, mimeType=audio/ac3, channels=2, sample_rate=48000, language=en, label=Commentary with Writers Ted Elliott and Terry Rossio, supported=YES
11-06 00:06:12.642 18798 18798 D EventLogger:     ]
11-06 00:06:12.642 18798 18798 D EventLogger:   ]
...
11-06 00:06:12.642 18798 18798 D EventLogger:   FfmpegAudioRenderer [
11-06 00:06:12.642 18798 18798 D EventLogger:     Group:0, adaptive_supported=N/A [
11-06 00:06:12.643 18798 18798 D EventLogger:       [X] Track:0, id=2, mimeType=audio/vnd.dts, channels=6, sample_rate=48000, language=en, label=D-Z0N3 HD AUDIO @ DTS 1510 Kbps Core English DTSHD-MA, supported=YES
11-06 00:06:12.643 18798 18798 D EventLogger:     ]
11-06 00:06:12.643 18798 18798 D EventLogger:   ]
...
```

The problem is that your device tries to play 2 different audios at the same time with 2 different audio decoders, which is not allowed in ExoPlayer, and even if it was, it should not happen in Jellyfin.

However, I don't quite understand how the extension should be used correctly, so I do not know how to fix it, so I leave the comment with my findings in case it helps the devs.

Relevant links:
- https://github.com/google/ExoPlayer/tree/release-v2/extensions/ffmpeg#using-the-module
- https://github.com/google/ExoPlayer/issues/2618

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -35,8 +35,8 @@ coil = "1.3.2"
 # Media
 androidx-media = "1.4.2"
 androidx-mediarouter = "1.2.5"
-exoplayer = "2.16.0"
-jellyfin-exoplayer-ffmpegextension = "2.16.0+1"
+exoplayer = "2.16.1"
+jellyfin-exoplayer-ffmpegextension = "2.16.1+1"
 playservices = "20.1.0"
 
 # Room

